resource/pick_languages.py

```diff
@@ -0,0 +1,11 @@
+from language_dict import SUPPORTED_LANGUAGES_SPACY
```

Comment:
Re: previous comment, we can remove this since the dynamic select will introduce too much UX delay

Reply:
I'll juste delete the file, doesn't need it anymore